browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
```

Comment:
A lot of client-side components use the mixin approach, while also being constructors, such as `Emitter`. (amongst others)

I think this is largely just a convention the `component/*` namespace has adopted, which is why it doesn't show up in node-land.